s3proxy/internal/router/object.go

```diff
@@ -0,0 +1,185 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package router
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/edgelesssys/constellation/v2/internal/logger"
+	"github.com/edgelesssys/constellation/v2/s3proxy/internal/crypto"
+)
+
+const (
+	// testingKey is a temporary encryption key used for testing.
+	// TODO (derpsteb): This key needs to be fetched from Constellation's keyservice.
+	testingKey = "01234567890123456789012345678901"
```

Comment:
I think we should address this right now. I'm very uneasy with variables that start with `testing` who not reside in a `_test.go` file. Especially if they are crypto keys. Especially if they are used right now.
Maybe since we aren't in a hurry with this feature and this is quite separate, let's wait for approval here and then create a PR with the KMS implementation to this branch?

Reply:
Addressed here: https://github.com/edgelesssys/constellation/pull/2398